user_guide/src/dsl/expressions.md

```diff
@@ -2,57 +2,93 @@
 
 The following is an expression:
 
-`pl.col("foo").sort().head(2)`
+<div class="tabbed-blocks">
+
+```python
+pl.col("foo").sort().head(2)
+```
+
+```rust,noplayground
+df.column("foo")?.sort(false).head(Some(2));
+```
+
+</div>
 
 The snippet above says:
 
 1. Select column "foo"
-1. Then sort the column
+1. Then sort the column (not in reversed order)
 1. Then take the first two values of the sorted output
 
 The power of expressions is that every expression produces a new expression, and that they
 can be *piped* together. You can run an expression by passing them to one of `Polars` execution contexts.
 
 Here we run two expressions by running `df.select`:
 
+<div class="tabbed-blocks">
+
 ```python
 df.select([
     pl.col("foo").sort().head(2),
     pl.col("bar").filter(pl.col("foo") == 1).sum()
 ])
 ```
 
-All expressions are ran in parallel, meaning that separate `Polars` expressions are **embarrassingly
-parallel**. Note that within an expression there may be more parallelization going on.
+```rust,noplayground
+df.select([
+   col("foo").sort(Default::default()).head(Some(2)),
+   col("bar").filter(col("foo").eq(lit(1))).sum(),
+]).collect()?;
+```
+
+</div>
+All expressions are run in parallel, meaning that separate `Polars` expressions are **embarrassingly parallel**. Note that within an expression there may be more parallelization going on.
 
 ## Expression examples
 
 In this section we will go through some examples, but first let's create a dataset:
 
+<div class="tabbed-blocks">
+
 ```python
 {{#include ../examples/expressions/dataset.py}}
 print(df)
 ```
 
+```rust,noplayground
+{{#include ../examples/expressions/expressions.rs:dataset}}
+```
+
+</div>
+
 ```text
 {{#include ../outputs/expressions/dataset.txt}}
 ```
 
 You can do a lot with expressions. They are so expressive that you sometimes have
-multiple ways to get the same results. To get a better feel for them let's go through some
-more examples.
+multiple ways to get the same results. To get a better feel for them let's go through some more examples.
+
```

Comment:
Maybe we should note that polars clones are super cheap because a `Series` is `Arc<ChunkedArray>`?